cfgov/regulations3k/jinja2/regulations3k/regulations3k-side-nav.html

```diff
@@ -15,15 +7,7 @@
 
 {# Extra classes needed for .content_sidebar #}
 {% block content_sidebar_modifiers -%}
+    {{ super() }}
     o-regs3k-sidebar
     content__flush-bottom
-    content__flush-top-on-small
-    content__flush-sides-on-small
-    {{ 'content__half-top-on-desk' if breadcrumb_items | length > 0 else '' }}
-{%- endblock %}
```

Comment:
@willbarton these three classes now [come from the parent template](https://github.com/cfpb/consumerfinance.gov/blob/86491284a97123b6bc164fbbd5c200b41d06894a/cfgov/v1/jinja2/v1/layouts/layout-side-nav.html#L9-L13) via the `super()` call.